src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/LoggingConstants.cs

```diff
@@ -10,20 +10,19 @@ internal class LoggingConstants
 
         // Error messages
         public const string RefreshFailedDueToAuthenticationError = "A refresh operation failed due to an authentication error.";
-        public const string RefreshFailedDueToKeyVaultError = "A refresh operation failed while resolving a Key Vault reference.";
         public const string RefreshFailedError = "A refresh operation failed.";
         public const string RefreshCanceledError = "A refresh operation was canceled.";
+        public const string RefreshFailedDueToKeyVaultError = "A refresh operation failed while resolving a Key Vault reference.";
 
         // Successful update messages
-        public const string RefreshConfigurationUpdatedSuccess = "Configuration reloaded for all selected key-values from endpoint: ";
-        public const string RefreshKeyValueChanged = "Change detected for key-value ";
-        public const string RefreshKeyValueUnchanged = "No change detected for key-value ";
-        public const string RefreshKeyValueSettingUpdated = "Value updated for key: ";
-        public const string RefreshKeyVaultSecretChanged = "Secret loaded from Key Vault for key-value ";
-        public const string RefreshKeyVaultSettingUpdated = "Value updated from Key Vault for key: ";
-        public const string RefreshFeatureFlagChanged = "Change detected for feature flag ";
-        public const string RefreshFeatureFlagsUnchanged = "No change detected for feature flags.";
-        public const string RefreshFeatureFlagValueUpdated = "Value updated for feature flag: ";
+        public const string RefreshConfigurationUpdatedSuccess = "Configuration reloaded.";
```

Comment:
I know we have a separate section on top for error messages, which are all logged as warnings. But for success, we sometimes print info and sometimes debug. It would be helpful to remember which statements are used for debug logs vs info logs. Can you split this into sections for info vs debug?